content/4_Gold/BFS.mdx

```diff
@@ -72,9 +97,17 @@ export const problems = {
 			['BFS'],
 			'usaco-695'
 		),
-		new Problem('Gold', "Bessie's Dream", '575', 'Hard', false, ['BFS']),
-		new Problem('Gold', "Replication", '1065', 'Hard', false, ['BFS']),
-		new Problem('Gold', 'A Pie for a Pie', '765', 'Very Hard', false, [], ''),
+		new Problem('Gold', "Bessie's Dream", '575', 'Hard', false, ['BFS'], ''),
```

Comment:
fix formatting here??

Reply:
I think it's fine if auto-formatter puts this all on a single line.